pages/result.js

```diff
@@ -84,153 +111,180 @@ export default function Index() {
+                      whiteSpace: 'nowrap',
+                    }}
                   >
-                    <Badge
-                      color="primary"
-                      badgeContent={
-                        <Typography sx={{ fontSize: '1rem', fontWeight: 600 }} component="span">
-                          {index + 4}
-                        </Typography>
-                      }
-                      anchorOrigin={{
-                        vertical: 'top',
-                        horizontal: 'left',
-                      }}
-                    >
-                      <Avatar alt={data.name} src={data.picture} sx={{ width: 48, height: 48, mr: 1 }} />
-                    </Badge>
-                    <Box
+                    <Typography
+                      noWrap
                       sx={{
-                        ml: 1,
-                        display: 'flex',
-                        flexDirection: 'column',
-                        textOverflow: 'ellipsis',
-                        overflow: 'hidden',
-                        whiteSpace: 'nowrap',
+                        display: 'inline-block',
+                        position: 'relative',
+                        fontSize: '1.6rem',
+                        fontWeight: 700,
                       }}
+                      color="text.primary"
+                    >
+                      {data.name}
+                    </Typography>
+                    <Typography
+                      sx={{ display: 'inline', fontSize: '1.2rem' }}
+                      component="span"
+                      variant="body2"
+                      color="text.primary"
                     >
-                      <Typography
+                      {generationCopy(data.generation)}
+                    </Typography>
+                    <Typography
+                      sx={{ display: 'inline', fontSize: '1.2rem', fontWeight: 600 }}
+                      component="span"
+                      variant="body2"
+                      color={!data.graduated ? 'green' : 'grey'}
+                    >
+                      {data.graduated ? 'Graduated' : 'Active'}
+                    </Typography>
+                  </Box>
+                  <Box
+                    sx={{
+                      ml: 2,
+                      display: 'flex',
+                      flex: 1,
+                      textOverflow: 'ellipsis',
+                      overflow: 'hidden',
+                      whiteSpace: 'nowrap',
+                      alignItems: 'center', // Mengatur vertikal tengah
+                      justifyContent: 'right', // Mengatur horizontal tengah
+                    }}
+                  >
+                    <Typography
+                      sx={{ fontSize: '1.2rem', fontWeight: 600 }}
+                      variant="body2"
+                    >
+                      {ordinalNumber(index + 1)} OSHI
+                    </Typography>
+                  </Box>
```

Comment:
I don't think this is necessary

the top member number is already shown on the badge that is on top of the member's avatar, showing another ordinal number just seems to be redundant information